add more docs for response helpers (#958)
solidjs/solid-docs: src/routes/solid-router/reference/response-helpers/revalidate.mdx

---
title: revalidate
---

> Revalidate will receive the either a cache key or an array of cache keys and invalidate those queries.

Reviewer: ```suggestion
Revalidate will receive either a cache key or an array of cache keys.
It will also invalidate those queries.
```

<Callout type="tip" name="Firing Queries Again">

> If you intend to fire the queries immediately again, check the [reload](/solid-router/reference/response-helpers/reload) helper.

Reviewer: ```suggestion
If you intend to re-fire the queries immediately, check the [reload](/solid-router/reference/response-helpers/reload) helper.
```
